package.json

```diff
@@ -20,7 +20,7 @@
     "prepublish": "npm run compile"
   },
   "peerDependencies": {
-    "node-sass": "^3.5.3"
+    "node-sass": "^4.0.0"
```

Comment:
At least in theory breakages should happen only in major versions. It may happen a bug is introduced by accident which causes a de facto breaking change but it should be way rarer than on majors.

That said, I'm fine with `>=3.5.3` either (until it's known 5.0.0 is incompatible or sth - but then a fix may land faster than the engines change anyway). The important part is to accept both `3.5.3` and `4.0.0`. :)